classes/PodsInit.php

```diff
@@ -446,47 +446,58 @@ public function setup_content_types( $force = false ) {
+				$cpt_labels['view']                  = pods_v( 'label_view', $post_type, '', true );
+				$cpt_labels['view_item']             = pods_v( 'label_view_item', $post_type, '', true );
+				$cpt_labels['all_items']             = pods_v( 'label_all_items', $post_type, '', true );
+				$cpt_labels['search_items']          = pods_v( 'label_search_items', $post_type, '', true );
+				$cpt_labels['not_found']             = pods_v( 'label_not_found', $post_type, '', true );
+				$cpt_labels['not_found_in_trash']    = pods_v( 'label_not_found_in_trash', $post_type, '', true );
+				$cpt_labels['parent']                = pods_v( 'label_parent', $post_type, '', true );
+				$cpt_labels['parent_item_colon']     = pods_v( 'label_parent_item_colon', $post_type, '', true );
+				$cpt_labels['archives']              = pods_v( 'label_archives', $post_type, '', true );
+				$cpt_labels['insert_into_item']      = pods_v( 'label_insert_into_item', $post_type, '', true );
+				$cpt_labels['uploaded_to_this_item'] = pods_v( 'label_uploaded_to_this_item', $post_type, '', true );
+				$cpt_labels['featured_image']        = pods_v( 'label_featured_image', $post_type, '', true );
+				$cpt_labels['set_featured_image']    = pods_v( 'label_set_featured_image', $post_type, '', true );
+				$cpt_labels['remove_featured_image'] = pods_v( 'label_remove_featured_image', $post_type, '', true );
+				$cpt_labels['use_featured_image']    = pods_v( 'label_use_featured_image', $post_type, '', true );
+				$cpt_labels['filter_items_list']     = pods_v( 'label_filter_items_list', $post_type, '', true );
+				$cpt_labels['items_list_navigation'] = pods_v( 'label_items_list_navigation', $post_type, '', true );
+				$cpt_labels['items_list']            = pods_v( 'label_items_list', $post_type, '', true );
 
 				// Supported
 				$cpt_supported = array(
-					'title'           => (boolean) pods_var( 'supports_title', $post_type, false ),
-					'editor'          => (boolean) pods_var( 'supports_editor', $post_type, false ),
-					'author'          => (boolean) pods_var( 'supports_author', $post_type, false ),
-					'thumbnail'       => (boolean) pods_var( 'supports_thumbnail', $post_type, false ),
-					'excerpt'         => (boolean) pods_var( 'supports_excerpt', $post_type, false ),
-					'trackbacks'      => (boolean) pods_var( 'supports_trackbacks', $post_type, false ),
-					'custom-fields'   => (boolean) pods_var( 'supports_custom_fields', $post_type, false ),
-					'comments'        => (boolean) pods_var( 'supports_comments', $post_type, false ),
-					'revisions'       => (boolean) pods_var( 'supports_revisions', $post_type, false ),
-					'page-attributes' => (boolean) pods_var( 'supports_page_attributes', $post_type, false ),
-					'post-formats'    => (boolean) pods_var( 'supports_post_formats', $post_type, false )
+					'title'           => (boolean) pods_v_sanitized( 'supports_title', $post_type, false ),
```

Comment:
When we convert to a boolean, sanitized isn't necessary so let's make these pods_v(), same when we convert to int too (just FYI)